index.js

```diff
@@ -199,7 +203,9 @@ const getCacheParams = ({
   let inputPaths, restoreKeys
 
   if (useYarn) {
-    inputPaths = [path.join(homeDirectory, '.cache', 'yarn')]
+    inputPaths = useYarnV1
+      ? [path.join(homeDirectory, '.cache', 'yarn')]
+      : [path.join(homeDirectory, '.yarn', 'berry', 'cache')]
```

Comment:
I ran `yarn info --cache` in repository which using yarn v4 and it says that the cache is in "$HOME/.yarn/berry/cache".